Classes/UT3GoliathTurret.uc

```diff
@@ -2,7 +2,8 @@
 // UT3GoliathTurret.uc
 // I don't think the sounds will initially work...
 // GreatEmerald, 2008, 2014
-// Copyright (c) 2012, 100GPing100 (visuals + fixed sounds)
+// Copyright © 2012, 2017 Luís 'zeluisping' Guimarães <zeluis.100@gmail.com> (visuals + fixed sounds)
+// HellDragon 2017 (I guess it helps identify anything I changed if there's a question about it later)
```

Comment:
You're missing `Copyright ©`, and no, you shouldn't actually list what you changed in the copyright line. Git keeps track of that instead. In case something goes wrong, we can `git blame` you ;)

Reply:
Oh....goody.....although I'll admit when something goes wrong I check to see if it was my fault and if yes I'm the first to say so usually

Reply:
You still haven't fixed this issue though!